<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #121212</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { 
    border: 0;
}
QTabBar::tab {
    background: transparent;
    color: lightgray;
    padding: 10px;
}
QTabBar::tab:selected {
    background: #1DB954;  /* Spotify green highlight */
    color: black;
    border-radius: 10px;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Generate</string>
     </attribute>
     <widget class="QPushButton" name="GenerateButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>500</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:lime</string>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QComboBox" name="MonthsDropdown">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>380</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:lime</string>
      </property>
      <item>
       <property name="text">
        <string>Whole Year</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="GenreCheckBox">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>300</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white</string>
      </property>
      <property name="text">
       <string>Genres</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Visuals</string>
     </attribute>
     <widget class="QLabel" name="ImageLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1081</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="Vis_MonthDropDown">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>100</y>
        <width>130</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background-color: #121212;
    border: 2px solid #1DB954;
    border-radius: 10px;
    padding: 5px 10px;
    color: white;
    font-size: 16px;
    font-family: 'Poppins', sans-serif;
}

QComboBox::drop-down {
    border: 0px;
}

QComboBox::down-arrow {
    image: url(down_arrow_white.png); /* optional: custom arrow icon */
    width: 14px;
    height: 14px;
}

QComboBox QAbstractItemView {
    background-color: #181818;
    border: 1px solid #1DB954;
    selection-background-color: #1DB954;
    selection-color: black;
    color: white;
    border-radius: 8px;
    padding: 5px;
}</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Vis_TypeDropDown">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>150</y>
        <width>130</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background-color: #121212;
    border: 2px solid #1DB954;
    border-radius: 10px;
    padding: 5px 10px;
    color: white;
    font-size: 16px;
    font-family: 'Poppins', sans-serif;
}

QComboBox::drop-down {
    border: 0px;
}

QComboBox::down-arrow {
    image: url(down_arrow_white.png); /* optional: custom arrow icon */
    width: 14px;
    height: 14px;
}

QComboBox QAbstractItemView {
    background-color: #181818;
    border: 1px solid #1DB954;
    selection-background-color: #1DB954;
    selection-color: black;
    color: white;
    border-radius: 8px;
    padding: 5px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>-10</y>
        <width>130</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #1DB954;
    font-size: 20px;
    font-weight: bold;
    font-family: 'Poppins', sans-serif;
    background-color: transparent;
    border-bottom: 2px solid #1DB954;
	padding-bottom: 5px;
    qproperty-alignment: AlignCenter;
}</string>
      </property>
      <property name="text">
       <string>Filters</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>1241</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>88</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IN PROGRESS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
